### ISSUE TITLE
Java AHRS: getDisplacement functions return wrong value

### ISSUE THREAD

**Issue by virtuald** [kauailabs/navxmxp#10]
They return `integrator.getVelocity*()` instead of `integrator.getDisplacementX()`.

**Comment by kauailabs** [comment]
Good catch; thanks, this is an issue in the backwards compatibility code (navX-MXP firmware v 2.0 moved the integration down into the navX-MXP microcontroller), and we missed that in testing.

This will get updated in a few days when we release the FRC 2016 version of the libraries.
